Document/0x06d-Testing-Data-Storage.md

```diff
@@ -851,38 +851,16 @@ func applicationWillEnterForeground(_ application: UIApplication) {
 }
 ```
 
-Objective-C:
-
-```objc
-@property (UIImageView *)backgroundImage;
-
-- (void)applicationDidEnterBackground:(UIApplication *)application {
-    UIImageView *myBanner = [[UIImageView alloc] initWithImage:@"overlayImage.png"];
-    self.backgroundImage = myBanner;
-    self.backgroundImage.bounds = UIScreen.mainScreen.bounds;
-    [self.window addSubview:myBanner];
-}
-
-- (void)applicationWillEnterForeground:(UIApplication *)application {
-    [self.backgroundImage removeFromSuperview];
-}
-```
-
-This sets the background image to `overlayImage.png` whenever the application is backgrounded. It prevents sensitive data leaks because `overlayImage.png` will always override the current view.
-
 ### Dynamic Analysis
 
-Navigate to an application screen that displays sensitive information, such as a username, an email address, or account details. Background the application by hitting the Home button on your iOS device. Connect to the iOS device and navigate to the following directory (which may be different for iOS versions below 14.2):
-
-`/var/mobile/Containers/Data/Application/$APP_ID/Library/SplashBoard/Snapshots/sceneID:$APP_NAME-default/`
-
-Older iOS versions may store the auto-generated screenshots under:
-
-`/var/mobile/Containers/Data/Application/$APP_ID/Library/Caches/Snapshots/`
-
-If the application caches the sensitive information in a screenshot, the app fails this test.
+Follow these steps using a jailbroken device or a non-jailbroken device after [repackaging the app with the Frida Gadget](0x06c-Reverse-Engineering-and-Tampering.md#dynamic-analysis-on-non-jailbroken-devices):
```

Comment:
You could just verify this visually from the screen, without a jailbroken iphone and without Frida.

Maybe we could add a difference between 'validating the testcase' and 'collecting the evidence' ?

Reply:
Agree & done ;)